mlflow/transformers.py

```diff
@@ -1446,9 +1446,16 @@ def _predict(self, data):
                 error_code=BAD_REQUEST,
             )
 
-        # Optional input preservation for specific pipeline types
+        # Optional input preservation for specific pipeline types. This is True (include raw
+        # formatting output), but if `include_prompt` is set to False in the `inference_config`
+        # option during model saving, excess newline characters and the fed-in prompt will be
+        # trimmed out from the start of the response.
         include_prompt = (
-            self.inference_config.pop("include_prompt", False) if self.inference_config else False
+            self.inference_config.pop("include_prompt", True) if self.inference_config else True
```

Comment:
Does `include_prompt` need to be popped out?

Reply:
unfortunately, yes. 
transformers pipeline execution includes validation for kwargs submitted. If we leave that inference kwarg entry in (by using `self.inference_config.get(...)`, we get:

E           ValueError: The following `model_kwargs` are not used by the model: ['include_prompt'] (note: typos in the generate arguments will also show up in this list)